torch/csrc/jit/tensorexpr/tensorexpr_init.cpp

```diff
@@ -451,7 +454,7 @@ void initTensorExprBindings(PyObject* module) {
   py::class_<LoopNest>(te, "LoopNest")
       .def(py::init<const std::vector<Tensor>&>())
       .def(py::init<const std::vector<Tensor>&, const std::vector<Tensor>&>())
-      .def(py::init([](StmtPtr s, const std::vector<BufHandle>& bufs) {
+      .def(py::init([](const StmtPtr& s, const std::vector<BufHandle>& bufs) {
         std::unordered_set<BufPtr> buf_nodes;
```

Comment:
Can you add a reserve here while you are editing this?

Reply:
Fixed